apisix/plugins/prometheus.lua

```diff
@@ -23,6 +23,13 @@ local plugin_name = "prometheus"
 local default_export_uri = "/apisix/prometheus/metrics"
 local schema = {
     type = "object",
+    properties = {
+        route_type = {
```

Comment:
`prefer_name` would be better?

Reply:
Thanks for the suggestion and I finally choose `prefer_type` since it could include `id` and `name`